src/mumble/Settings.h

```diff
@@ -431,10 +431,15 @@ struct Settings {
 	QByteArray qbaPTTButtonWindowGeometry = {};
 
 	// Network settings
-	bool bTCPCompat   = false;
-	bool bReconnect   = true;
-	bool bAutoConnect = false;
-	bool bQoS         = true;
+	bool bTCPCompat              = false;
```

Comment:
I think a better name for this setting would be e.g. `bUdpDisabled`
That way it would be clearer that it is related to the same mechanism as the other (new) settings

Reply:
Does it mean also mean that the strings like "Force TCP Mode" also must be changed to "Disable UDP"?

Reply:
Yeah, I think from a consistency point of view that would be best.

Note that with the new settings framework, renaming a setting is really easy. Just prepend the new name to the list here:
https://github.com/mumble-voip/mumble/blob/c87fb2b4ed76fe643a24eb1100f4a14c18ed0306/src/mumble/SettingsKeys.h#L91
```cpp
const SettingsKey RESTRICT_TO_TCP_KEY               = { "disable_udp", "restrict_to_tcp" };
```
(and maybe also rename the variable's name accordingly)

The `SettingsKey` implementation should then automatically take care of creating a migration path from the old setting name to the new one (should be tested though).

Reply:
Does it mean there won't be anything that says "Force TCP mode"? And instead now, are we completely calling everything as "UDP disabled mode"?

Reply:
> Does it mean there won't be anything that says "Force TCP mode"? And instead now, are we completely calling everything as "UDP disabled mode"?

Yes that is what I would have imagined. That seems like the most consistent choice :thinking: 

> I see that "tcponly" is replaced by "disable_udp".

Yeah that's the expected behavior. Well actually `tcponly` was the key that was used in the old INI format of the settings. With the change to the JSON format, that setting should be called `restrict_to_tcp` and with your changes this should then be replaced by `disable_udp`.